yarl/_url.py

```diff
@@ -904,7 +904,7 @@ def _query_seq_pairs(cls, quoter, pairs):
     def _query_var(v):
         if isinstance(v, str):
             return v
-        if type(v) is int:  # no subclasses like bool
+        if isinstance(v, int) and not isinstance(v, bool):  # no subclasses like bool
```

Comment:
This will probably make it slower so I'll leave it to @asvetlov to decide. But you'd probably have to add tests for this (both negative and positive) and find other places in code that use the same idiom.

Also, what if such `int` subclass implements a weird `__str__()` method that returns something odd? This would create hard-to-debug issues in the calling code, especially if your colleagues don't know YARL internals. Is it really worth it? This could probably be mitigated by an extra check for `v.__str__ is int.__str__` but isn't it overkill?